decidim-core/app/views/layouts/decidim/shared/_drawer.html.erb

```diff
@@ -0,0 +1,15 @@
+<div data-drawer>
+  <div>
+    <div data-drawer-close><%= icon "close-line", class: "w-5 h-5 text-secondary fill-current" %></div>
+    <% if local_assigns[:prev].present? %>
+      <div data-drawer-prev><%= icon "arrow-right-s-line", class: "w-8 h-8 text-secondary fill-current" %></div>
```

Comment:
They do not do nothing because the `locals` variable has strings defined for such attributes. Since @entantoencuanto has been working on Turbo, using accountability as base url, this will change shortly, so we can let it go.